pkg/core/native/nonfungible.go

```diff
@@ -81,7 +81,7 @@ func newNonFungible(name string, id int32, symbol string, decimals byte) *nonfun
 	n.AddMethod(md, desc)
 
 	desc = newDescriptor("ownerOf", smartcontract.Hash160Type,
-		manifest.NewParameter("tokenId", smartcontract.ByteArrayType))
+		manifest.NewParameter("tokenid", smartcontract.ByteArrayType))
```

Comment:
But why? States will be different in this case. Let's leave it as in C# node (and change ours NFT standard) and wait for the C# devs to resolve the issue with the standard. We may create a separate issue in neo-go for this.